### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Poussinou**
Hi,

This small PR adds a button to get your app on F-Droid 👍 It also removes the link to IzzyOnDroid, as your app is now in the official F-Droid repo.

It also adds a link to get the latest APK from the Releases Section.

You can now eventually add the [fdroid](https://github.com/topics/fdroid) and/or [f-droid](https://github.com/topics/f-droid) tags that will appear on the right side of the repo.

### CODE REVIEW

**Comment by VegaBobo** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -19,13 +19,12 @@ Google GSIs: https://developer.android.com/topic/generic-system-image/releases
...
 
 ## Downloads
-Builds are available at:
 
-| DSU Sideloader |
-|----------------|
-| <a href="https://github.com/VegaBobo/DSU-Sideloader/releases"><img src="https://img.shields.io/github/v/release/VegaBobo/DSU-Sideloader?label=GitHub" /></a> |
-| <a href="https://apt.izzysoft.de/fdroid/index/apk/vegabobo.dsusideloader"><img src="https://img.shields.io/endpoint?url=https://apt.izzysoft.de/fdroid/api/v1/shield/vegabobo.dsusideloader" /></a> |
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
+     alt="Get it on F-Droid"
```
Comment (gibbsbradley66-sketch) [inline README.md:24]: Alt=

File: README.md [hunk 2]
```diff
@@ -19,13 +19,12 @@ Google GSIs: https://developer.android.com/topic/generic-system-image/releases
 **Remember to use GSIs compatible with your architeture, vndk implementation..*
 
 ## Downloads
-Builds are available at:
 
-| DSU Sideloader |
-|----------------|
-| <a href="https://github.com/VegaBobo/DSU-Sideloader/releases"><img src="https://img.shields.io/github/v/release/VegaBobo/DSU-Sideloader?label=GitHub" /></a> |
```
Comment (gibbsbradley66-sketch) [inline README.md:26]: <a

File: README.md [hunk 3]
```diff
@@ -19,13 +19,12 @@ Google GSIs: https://developer.android.com/topic/generic-system-image/releases
...
-| <a href="https://apt.izzysoft.de/fdroid/index/apk/vegabobo.dsusideloader"><img src="https://img.shields.io/endpoint?url=https://apt.izzysoft.de/fdroid/api/v1/shield/vegabobo.dsusideloader" /></a> |
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
```
Comment (gibbsbradley66-sketch) [inline README.md:23]: <+

File: README.md [hunk 4]
```diff
@@ -19,13 +19,12 @@ Google GSIs: https://developer.android.com/topic/generic-system-image/releases
 **Remember to use GSIs compatible with your architeture, vndk implementation..*
 
 ## Downloads
-Builds are available at:
 
-| DSU Sideloader |
-|----------------|
-| <a href="https://github.com/VegaBobo/DSU-Sideloader/releases"><img src="https://img.shields.io/github/v/release/VegaBobo/DSU-Sideloader?label=GitHub" /></a> |
-| <a href="https://apt.izzysoft.de/fdroid/index/apk/vegabobo.dsusideloader"><img src="https://img.shields.io/endpoint?url=https://apt.izzysoft.de/fdroid/api/v1/shield/vegabobo.dsusideloader" /></a> |
```
Comment (gibbsbradley66-sketch) [inline README.md:27]: Img

**Review by gibbsbradley66-sketch — CHANGES_REQUESTED** [review]

**Review by gibbsbradley66-sketch — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (gibbsbradley66-sketch) [reply README.md:27]: Src

Comment (gibbsbradley66-sketch) [reply README.md:27]: Src

Comment (gibbsbradley66-sketch) [reply README.md:27]: Src

**Pull request merged**